### PR TITLE
[URGENT] DEVELOPER-2987 Added salary survey landing page

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

File: promotions/developers-salary-survey-2016.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// Microsite title
+### Curious how you stack up compared to your colleagues? We were curious too.
+
+// Microsite subtitle
+#### So we worked with Evans Data to learn more. Evans Data reached out to 400 enterprise developers to learn what software developers make and what impacts those salary ranges.
+
+
+*In this 2016 Salary Survey Report learn:*
+
+* Developers average salaries
```
Comment (paulrobinson) [inline promotions/developers-salary-survey-2016.adoc]: Developers -> Developers'

File: promotions/developers-salary-survey-2016.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* How education, company, certifications, and more impact earning potential
+* Why language abilities and developer program membership make a difference
+* 5 key things you can do to increase your income potential
+
+Download the Evans Data Developers Salary Survey Recap to learn more.
```
Comment (paulrobinson) [inline promotions/developers-salary-survey-2016.adoc]: should this be quoted or italicized:

      Evans Data Developers Salary Survey Recap

File: promotions/developers-salary-survey-2016.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Want to dive into the research even further? Download the full report.
+
+[.button]
+link:#{site.download_manager_file_base_url}/download-manager/file/Developers_Salary_Report_2016.pdf[DOWNLOAD NOW]
```
Comment (paulrobinson) [inline promotions/developers-salary-survey-2016.adoc]: Produces:

https://developers.stage.redhat.com/download-manager/file/download-manager/file/Developers_Salary_Report_2016.pdf

Should be:

https://developers.stage.redhat.com/download-manager/file/Developers_Salary_Report_2016.pdf

**Comment by paulrobinson** [comment]
Have you checked that the Download JS is working? I hacked the link I got back from the download manager to: 

      http://developers-pr.stage.redhat.com/pr/1102/build/1180/promotions/developers-salary-survey-2016/?tcUser=Paul+Robinson&tcWhenSigned=November+16%2C+2015&tcWhenEnds=November+15%2C+2016&tcEndsIn=184&tcDuration=365&tcDownloadFileName=Developers_Salary_Report_2016.pdf&tcRedirect=5000&tcSrcLink=https%3A%2F%2Fwww.jboss.org%2Fdownload-manager%2Fcontent%2Forigin%2Ffiles%2Fsha256%2F57%2F57d076b6729a374aa9b24d8fa7733ab42907bc6394ee642929e13d72045c822b%2FDevelopers_Salary_Report_2016.pdf&p=Developer+Salary+Report+2016&pv=developer+salary+report+2016&tcDownloadURL=https%3A%2F%2Faccess.cdn.redhat.com%2Fcontent%2Forigin%2Ffiles%2Fsha256%2F57%2F57d076b6729a374aa9b24d8fa7733ab42907bc6394ee642929e13d72045c822b%2FDevelopers_Salary_Report_2016.pdf%3F_auth_%3D1463409019_f7f8b51b16d8e898bbcfd7be80fdb341

However, nothing happened when the page loaded. I don't see any errors in the console either.

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by adelasofia** [comment]
Approved by Lincoln Baxter

**Comment by adelasofia** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

**Pull request merged**